skfda/preprocessing/dim_reduction/projection/_fpca.py

```diff
@@ -397,3 +397,39 @@ def fit_transform(self, X, y=None, **fit_params):
         """
         self.fit(X, y)
         return self.transform(X, y)
+
+    def get_component_perturbations(self, X, index=0, multiple=30):
```

Comment:
Why is this a method of FPCA?

Reply:
In this case just to access the components. It can also be a static method outside of FPCA. Making it part of FPCA just makes it easier for the user, in my opinion.

Reply:
I think that is better to make a function in the visualization module, which accepts as parameters the mean and an iterable of components, making a subplot for each component.

Reply:
skfda.exploratory.visualization fpca.py?
I agree with you about making a method to plot the diagram for each component, I'm not sure if this is the location where I should put it.

Reply:
That location looks good to me.

Reply:
Done